front/src/modules/pathfinding/hooks/usePathfinding.ts

```diff
@@ -229,8 +287,13 @@ export const usePathfinding = (
             pathfindingResult.status === 'failure' &&
             pathfindingResult.failed_status === 'pathfinding_not_found' &&
             pathfindingResult.error_type === 'incompatible_constraints';
-
-          if (pathfindingResult.status === 'success' || incompatibleConstraintsCheck) {
+          if (
+            pathfindingResult.status === 'failure' &&
+            pathfindingResult.failed_status === 'pathfinding_input_error' &&
+            pathfindingResult.error_type === 'invalid_path_items'
+          ) {
+            handleInvalidPathItems(pathfindingResult, pathSteps, dispatch, pathfindingDispatch, t);
```

Comment:
Don't give the pathfindingResult, but directly the invalid pathSteps: `pathfindingResult.items`
```suggestion
            handleInvalidPathItems(pathSteps, pathfindingResult.items);
```

Reply:
done